mithril-aggregator/src/runtime/runner.rs

```diff
@@ -876,281 +943,161 @@ pub mod tests {
     }
 
     #[tokio::test]
-    async fn test_get_current_non_certified_open_message_should_create_new_open_message_for_mithril_stake_distribution_if_none_exists(
+    async fn test_get_current_non_certified_open_message_should_create_new_open_message_if_none_exists(
```

Comment:
The precision that a the type of the open message is a mithril stake distribution matters here.
But maybe two concerns were mixed: `get_current_non_certified_open_message` should try to create open message by following the order of the signed entity type list returned by `config.list_allowed_signed_entity_types`.

This matter since this allow us to:
1. Limit the list of signed entity types handled by the aggregator (only those listed should be used to create open message, an existing type but not listed should not be used).
2. To give the order that the aggregator will follow to sign entities (ie: that's how we signed the Mithril Stake distribution fist in an epoch, before all other types).

The updated test is fine but we should add one or two tests to check this behavior.

Reply:
Agree with that. 
It's clear with this refactor that we do not check signed entity type. It wasn't verify before but it wasn't obvious